actions/ansible-docs-build-diff/action.yml

```
@@ -75,6 +78,16 @@ outputs:
 runs:
   using: composite
   steps:
+    - name: Delete files that should not be included in the diff
+      id: delete
+      shell: bash
+      run: |
+        echo "::group::Deleting files from ${{ inputs.build-html-a }}"
+        find "${{ inputs.build-html-a }}" \( -name '*.js' -or -name '*.inv' \) -delete -print
+        echo "::endgroup::"
+        echo "::group::Deleting files from ${{ inputs.build-html-b }}"
+        find "${{ inputs.build-html-b }}" \( -name '*.js' -or -name '*.inv' \) -delete -print
+        echo "::endgroup::"
```

```suggestion
        echo "::endgroup::"
        
```
nit: extra newline between steps